en/grouping.html

```diff
@@ -1447,6 +1447,15 @@ <h2 id="nested-groups">Nested Groups</h2>
 <h2 id="structured-grouping">Structured grouping</h2>
 <p>
 Structured grouping is nested grouping over an array of structs or maps.
+In this case, each array element is treated as a sub-document and may
+be grouped separately. See the reference for grouping on
```

Comment:
I added https://github.com/vespa-engine/documentation/pull/3796 also today, so we might look at linking all this